evap/evaluation/templates/startpage_button.html

```diff
@@ -0,0 +1,13 @@
+{% if user.startpage == page %}
+    <div data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'This is your startpage' %}">
+        <button type="button" class="btn {{ btnClass }} btn-light" disabled><span class="fas fa-house"></span></button>
+    </div>
+{% else %}
+    <form id="startpage-form" class="d-flex" method="post" action="{% url 'evaluation:set_startpage' %}">
+        {% csrf_token %}
+        <input name="page" type="hidden" value="{{ page }}">
+        <button id="makeStartPageButton" type="submit" class="btn {{ btnClass }} btn-light" onclick="setSpinnerIcon('span-set-startpage')" data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'Make this my startpage' %}">
```

Comment:
This seems to be unused:
```suggestion
        <button type="submit" class="btn {{ btnClass }} btn-light" onclick="setSpinnerIcon('span-set-startpage')" data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'Make this my startpage' %}">
```